airbyte-integrations/connectors/source-northpass-lms/metadata.yaml

```
@@ -0,0 +1,35 @@
+data:
+  allowedHosts:
+    hosts:
+      - api.northpass.com/v2
```

```suggestion
      - api.northpass.com
```

Just the hostname is fine, let's keep it simple.